docker-compose.yml

```diff
@@ -9,7 +9,7 @@ db:
 base:
   command: /bin/true
   build: .
-console:
+logplex_console:
```

Comment:
For not-so-important reason of avoiding potential confusion with running 'docker-compose run console' expecting it to behave as it does *without* volumes mounted. Can revert if you want.